sdk/servicebus/service-bus/README.md

```diff
@@ -55,18 +55,45 @@ To use this client library in the browser, first you need to use a bundler. For
 
 ### Authenticate the client
 
-Interaction with Service Bus starts with an instance of the [ServiceBusClient][sbclient] class.
+Interaction with Service Bus starts with an instance of the [ServiceBusClient][sbclient] class. You can
+authenticate to Service Bus using a connection string or using an Azure Active Directory credential.
 
-You can instantiate this class using its constructors:
+#### Using a connection string
 
-- [Create using a connection string][sbclient_constructor]
-  - This method takes the connection string to your Service Bus instance. You can get the connection string
-    from the Azure portal.
-- [Create using a TokenCredential][sbclient_tokencred_overload]
-  - This method takes the host name of your Service Bus instance and a credentials object that you need
-    to generate using the [@azure/identity](https://www.npmjs.com/package/@azure/identity)
-    library. The host name is of the format `name-of-service-bus-instance.servicebus.windows.net`.
-    If you're using an own implementation of the `TokenCredential` interface against AAD, then set the "scopes" for service-bus to be `["https://servicebus.azure.net//user_impersonation"]` to get the appropriate token.
+This method takes the connection string to your Service Bus instance. You can get
+the connection string from the Azure portal.
+
+```javascript
+const { ServiceBusClient } = require("@azure/service-bus");
+
+const serviceBusClient = new ServiceBusClient(connectionString);
+```
+
+More information about this constructor is available in the [API documentation][sbclient_constructor].
+
+#### Using an Azure Active Directory Credential
+
+Authentication with Azure Active Directory uses the [Azure Identity library][azure_identity].
+
+The example below uses the [DefaultAzureCredential][defaultazurecredential], one of many
+available credential providers from the `@azure/identity` library.
+
+```javascript
+const { ServiceBusClient } = require("@azure/service-bus");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+const endpoint = "<name-of-service-bus-instance>.servicebus.windows.net";
+const credential = new DefaultAzureCredential();
+const serviceBusClient = new ServiceBusClient(endpoint, credential);
+```
+
+    NOTE: If you're using your own implementation of the `TokenCredential` interface
+    against AAD, then set the "scopes" for service-bus to the following to get
+    the appropriate token:
+
+    ["https://servicebus.azure.net//user_impersonation"]
```

Comment:
This part is being rendered weirdly as if the whole thing is a code block...

Reply:
Apparently what I wanted to use was 'blockquote' (using >).

Fixed, should look better now.